src/payment/strategies/klarnav2/klarnav2-payment-strategy.ts

```diff
@@ -0,0 +1,185 @@
+import { includes } from 'lodash';
+
+import { Address } from '../../../address';
+import { BillingAddress } from '../../../billing';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError, MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { RemoteCheckoutActionCreator } from '../../../remote-checkout';
+import { PaymentMethodCancelledError, PaymentMethodInvalidError } from '../../errors';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import KlarnaPayments, { KlarnaAddress, KlarnaAuthorizationResponse, KlarnaLoadResponse, KlarnaUpdateSessionParams } from './klarna-payments';
+import KlarnaV2ScriptLoader from './klarnav2-script-loader';
+
+export default class KlarnaV2PaymentStrategy implements PaymentStrategy {
+    private _klarnaPayments?: KlarnaPayments;
+    private _unsubscribe?: (() => void);
+    private _supportedEUCountries = ['AT', 'DE', 'DK', 'FI', 'GB', 'NL', 'NO', 'SE', 'CH'];
+
+    constructor(
+        private _store: CheckoutStore,
+        private _orderActionCreator: OrderActionCreator,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator,
+        private _remoteCheckoutActionCreator: RemoteCheckoutActionCreator,
+        private _klarnav2ScriptLoader: KlarnaV2ScriptLoader
+    ) {}
+
+    initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        return this._klarnav2ScriptLoader.load()
+            .then(klarnaPayments => { this._klarnaPayments = klarnaPayments; })
+            .then(() => {
+                this._unsubscribe = this._store.subscribe(
```

Comment:
This whole `subscribe` block seems to be doing nothing so you can probably just remove it. As a result, you can also remove the `_unsubscribe` call in `deinitialize`.